README.md

```diff
@@ -2,176 +2,103 @@
     <img src="./docs/src/logo.svg" width="400">
 </p>
 
-<h3 align="center">
-Dataflow Oriented Robotic Architecture ⚡
-</h3>
 
----
 
-`dora` goal is to be a low latency, composable, and distributed data flow.
+<h2 align="center">
+  <a href="https://dora.carsmos.ai">Website</a>
+  |
+  <a href="https://dora.carsmos.ai/docs/api/python-api">Python API</a>
+  -
+  <a href="https://docs.rs/dora-node-api/latest/dora_node_api/">Rust API</a>
+  |
+  <a href="https://dora.carsmos.ai/docs/guides/">Guide</a>
+  |
+  <a href="https://github.com/orgs/dora-rs/discussions">Discussion</a>
+</h2>
+
+<div align="center">
+  <a href="https://github.com/dora-rs/dora/actions">
+    <img src="https://github.com/dora-rs/dora/workflows/CI/badge.svg" alt="Build and test"/>
+  </a>
+  <a href="https://crates.io/crates/dora-rs">
+    <img src="https://img.shields.io/crates/v/dora_node_api.svg"/>
+  </a>
+  <a href="https://docs.rs/dora-node-api/latest/dora_node_api/">
+    <img src="https://docs.rs/dora-node-api/badge.svg" alt="rust docs"/>
+  </a>
+  <a href="https://pypi.org/project/dora-rs/">
+    <img src="https://img.shields.io/pypi/v/dora-rs.svg" alt="PyPi Latest Release"/>
+  </a>
+</div>
 
-This project is in early development, and many features have yet to be implemented with breaking changes. Please don't take for granted the current design.
+---
 
-`dora` primary support is with `Linux` as it is the primary OS for both Cloud and small computers. If you wish to use `dora` with another OS, please compile from source.
+## Why dora-rs?
 
----
-## 📖 Documentation
+In 2023, AI is booming! Robotic framework however hasn't changed much in years... This is why we create dora-rs! dora-rs is a new robotic framework that brings modernity into robotic application.
 
-The documentation can be found here: [https://dora-rs.github.io/dora/](https://dora-rs.github.io/dora/) 
+dora-rs can already show impressive performance, up to 17x speedup compared to ROS2! This is the result of using our own shared memory server and Apache Arrow to achieve zero copy!
```

Comment:
This speedup is for dora Python compared to ROS2 Python, right? So maybe we should mention Python here?

Reply:
Yeah absolutely!